_layouts/home.html

```diff
@@ -18,7 +18,8 @@ <h1>
           digital rights | civic transparency | data liberation
         </p>
         <p>
-        <a href="https://paypal.me/lucyparsonslabs"><span class="donate">DONATE</span></a>
+        <a href="https://paypal.me/lucyparsonslabs"><script src="https://donorbox.org/install-popup-button.js" type="text/javascript" defer></script></a>
```

Comment:
i think we can drop the `<a href="https://paypal.me/lucyparsonslabs"></a>` here unless you want a fallback for folks without JS?